tolerance-policies/apps/monitor/predict_many.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+while :; do
+    RESPONSE=$(curl -s localhost:8080/cpu/predict)
+    TRIMMED=$(echo ${RESPONSE} | sed 's/\(.\{50\}\).*/\1...\}/' )
```

Comment:
Let's escape these to promote good practices

```suggestion
    RESPONSE="$(curl -s localhost:8080/cpu/predict)"
    TRIMMED="$(echo ${RESPONSE} | sed 's/\(.\{50\}\).*/\1...\}/' )"
```